src/MarianInterface.cpp

```diff
@@ -4,29 +4,23 @@
 #include "3rd_party/bergamot-translator/src/translator/parser.h"
 #include "3rd_party/bergamot-translator/src/translator/response.h"
 #include "3rd_party/bergamot-translator/3rd_party/marian-dev/src/3rd_party/spdlog/spdlog.h"
+#include "translator/translation_model.h"
+#include <future>
```

Comment:
future is no longer part of the bergamot translator interface so you need to include it?

Reply:
I'm using `std::promise` to wait for the `AsyncService::translate()` call to finish. I assumed that none of the other headers would provide it anymore.